core/shared/src/main/scala/coursier/util/Parse.scala

```diff
@@ -146,12 +153,12 @@ object Parse {
     *  or
     *   org:name:version:config,attr1=val1,attr2=val2
     *
-    *  Currently only "classifier" attribute is used, and others are ignored.
+    *  Currently only the "classifier" attribute is used, and others are ignored.
```

Comment:
Docstring needs update since 'classifier' and 'url' are in use. The return value is also changed.